src/components/molecules/message/message.scss

```
@@ -0,0 +1,37 @@
+@use "../../variables" as *;
+
+.message {
+  text-align: center;
+  padding: 32px 20px;
+
+  & > .title {
```

子結合子で書いている理由はありますか？（個人的に子結合子より子孫結合子の方がよく使われているイメージがあるので気になりました。）

子孫結合子で書くとしたら下のようになります。現状でも全く問題はないのでお好みで修正お願いします。

子結合子： https://developer.mozilla.org/ja/docs/Web/CSS/Child_combinator
子孫結合子： https://developer.mozilla.org/ja/docs/Web/CSS/Descendant_combinator
```suggestion
.message {
  text-align: center;
  padding: 32px 20px;

   .title {
```

---

「Storybookの使い方、Sassについて」を見ながら書いていたのですが理解が曖昧でした。。。
ありがとうございます！

---

messageの中のtitleセレクタ
という意味で理解しやすくしたかったのでこちらにしました！
ありがとうございます。